tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,26 @@
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+gpio_t DASHBOARD_POWER = PB0;
+gpio_t SERVICE_SECTION_POWER = PB1;
+gpio_t THROTTLE_POWER = PB2;
+
+/*
+ * Power on the given device
+ */
+void power_device(gpio_t pin);
```

Comment:
I wouldn't name the argument `pin`, I'd probably name it `device` since that is what it refers to. I would also add a description in the comment about what the argument _is_. The idea is that someone should be able to read the docstring and understand what the function does and how to use it, and part of that is understanding what the arguments mean.